src/psyclone/gocean1p0.py

```diff
@@ -1586,15 +1585,17 @@ def name(self):
 
     def dereference(self, fld_name):
         '''Returns a Fortran string to dereference a grid property of the
-        specified field. E.g."name%grid%dx". The stored value of
-        self._dereference_name is a format string, where {0} represents
-        the field name.
+        specified field. It queries the current config file settings for
+        getting the proper dereference string, which is a format string
+        where {0} represents the field name.
 
```

Comment:
Please document the `fld_name` argument to this routine.

Reply:
Done.